examples/todo.rs

```diff
@@ -30,16 +31,23 @@ use rustful::StatusCode;
 
 //Helper for setting a status code and then returning from a function
 macro_rules! or_abort {
-    ($e: expr, $response: expr, $status: expr) => (
+    ($e: expr, $response: ident, $error_message: ident) => (
         if let Some(v) = $e {
             v
         } else {
-            $response.set_status($status);
+            $response.set_status(StatusCode::BadRequest);
+            $response.headers_mut().set(ContentType(content_type!(Text / Plain; Charset = Utf8)));
+            $response.send($error_message);
             return
         }
     )
 }
 
+type ErrorMessage = &'static str;
```

Comment:
I don't think a type alias is as helpful here as in other places. `&'static str` should be a well known type in it self and aliasing it may be obscuring. It's also not used anywhere else in  the code, as far as I can see.